pkgs/by-name/cu/cura-appimage/package.nix

```
@@ -0,0 +1,128 @@
+{
+  lib,
+  stdenv,
+  stdenvNoCC,
+  fetchurl,
+  writeScriptBin,
+  appimageTools,
+  copyDesktopItems,
+  makeDesktopItem,
+  nix-update-script,
+  wrapGAppsHook,
```

```suggestion
  wrapGAppsHook3,
```
Shouldn't use aliases in package definitions.
LGTM otherwise